Sources/SwiftBuildSupport/SwiftBuildSystem.swift

```diff
@@ -687,6 +723,32 @@ public final class SwiftBuildSystem: SPMBuildCore.BuildSystem {
         return settings
     }
 
+    private static func constructAPIDigesterSettingsOverrides(from digesterMode: BuildParameters.APIDigesterMode?) -> [String: String] {
```

Comment:
suggestion: although this function is straightforward, consider writing automated tests to values the expected return value.